jumpscale/packages/weblibs/package.py

```diff
@@ -11,7 +11,15 @@ def install(self):
         """Called when package is added
         """
         if not j.sals.fs.exists(j.sals.fs.join_paths(self.path, "jumpscaleX_weblibs")):
-            j.tools.git.clone_repo(url=self.url, dest=self.path, branch_or_tag=self.branch)
+            retries = 5
```

Comment:
5 is too much let's make it 3

Reply:
what happens if it failed and the dir got created and not a real git repo?

Reply:
git doesn't create the repo file unless it's completed. no ?

Reply:
tried cloning with wrong branch and anther with no internet connection and no dirs created :+1: